performance-es.md

```diff
@@ -12,11 +12,48 @@ Este libro está dividido en diferentes secciones:
    * Temas básicos de Ciencias de la Computación
 2. Consejos para escribir software eficiente.
    * Secciones específicas de Go sobre cómo obtener lo mejor del lenguaje
-3. Consejos avanzados para escribir * software realmente * eficiente
+3. Consejos avanzados para escribir *software realmente* eficiente
    * Para cuando tu código optimizado no sea lo suficientemente eficiente.
 
 Podemos resumir estas tres secciones como:
 
 1. "Sé razonable"
 2. "Sé deliberado"
 3. "Sé peligroso"
+
+# Cuando y donde optimizar
+
+Escribo esta sección primero porque es el paso más importante. Deberías estar haciendo esto?
+
+Todas las optimizaciones tienen un costo. Generalmente, este costo es expresado en terminos de complejidad de codigo o carga cognitiva -- un codigo optimizado es rara vez más simple que una versión sin optimizar.
+
+Pero hay otro aspecto que llamaré la economía de la optimización. Como programador, tu tiempo es valioso. Existe el costo oportuno de algun otro proyecto en el que puedas estar trabajando, que `bug` puedes arreglar, cual mejora puedeas agregar. Optimizar cosas es divertido, pero no siempre es la tarea correcta para escoger. El rendimiento de un programa es una característica, pero también lo es terminarlo y cuan correcto está hecho.
+
+Escoge lo más importanted en que debas trabajar. A veces, eso no significa una optimización de CPU, sino una de experiencia de usuario. Algo tán simple como agregar una barra de progreso, o modificar la página para que sea más veloz mediante cálculos en el fondo después de reproducir la página.
+
+Algunas veces esto será obvio: un reporte de cada hora que demora 3 horas en completar es menos ùtil que uno que termina en menos de una hora.
+
+Solo porque algo es fácil de optimizar no significa que valga la pena hacerlo. Ignorar lo simple es una estrategia de desarrollo válida.
+
+Considera esto como optimizar *tu* tiempo.
```

Comment:
```suggestion
Considera esto como optimizar *tú* tiempo.
```

Reply:
I think here is without tilde, since is the possessive pronoun in this case, referring to `your` time, not to `you`